test/system/250-systemd.bats

```diff
@@ -36,41 +40,23 @@ function service_setup() {
 
     # Also test enabling services (see #12438).
     run systemctl enable "$SERVICE_NAME"
-    if [ $status -ne 0 ]; then
-        die "Error enabling systemd unit $SERVICE_NAME, output: $output"
-    fi
+    assert $status -eq 0 "Error enabling systemd unit $SERVICE_NAME: $output"
 
     run systemctl start "$SERVICE_NAME"
-    if [ $status -ne 0 ]; then
-        die "Error starting systemd unit $SERVICE_NAME, output: $output"
-    fi
+    assert $status -eq 0 "Error starting systemd unit $SERVICE_NAME: $output"
 
     run systemctl status "$SERVICE_NAME"
-    if [ $status -ne 0 ]; then
-        die "Non-zero status of systemd unit $SERVICE_NAME, output: $output"
-    fi
+    assert $status -eq 0 "systemctl status $SERVICE_NAME: $output"
 }
 
 # Helper to stop a systemd service running a container
 function service_cleanup() {
     local status=$1
     run systemctl stop "$SERVICE_NAME"
-    if [ $status -ne 0 ]; then
-        die "Error stopping systemd unit $SERVICE_NAME, output: $output"
-    fi
+    assert $status -eq 0 "Error stopping systemd unit $SERVICE_NAME: $output"
 
     run systemctl disable "$SERVICE_NAME"
-    if [ $status -ne 0 ]; then
-        die "Error disbling systemd unit $SERVICE_NAME, output: $output"
-    fi
-
-    if [[ -z "$status" ]]; then
-        run systemctl is-active "$SERVICE_NAME"
-        if [ $status -ne 0 ]; then
-            die "Error checking stauts of systemd unit $SERVICE_NAME, output: $output"
-        fi
-        is "$output" "$status" "$SERVICE_NAME not in expected state"
-    fi
```

Comment:
I have no idea what this was intended to do, but it's a NOP, so I've removed it.